src/index.ts

```diff
@@ -1,56 +1,14 @@
-import { CHAINS } from './generated/chains';
-import { HardhatConfigNetworks, HardhatEtherscanNetworks } from './types';
-
-export * from './types';
+import * as hardhat from './hardhat';
 
 // NOTE: the following file is generated with the generate-chains.ts script
 export { CHAINS } from './generated/chains';
 
-export function hardhatConfigNetworks(): HardhatConfigNetworks {
-  return CHAINS.reduce((networks, chain) => {
-    networks[chain.alias] = {
-      accounts: { mnemonic: '' },
-      chainId: Number(chain.id),
-      url: chain.providerUrl,
-    };
-    return networks;
-  }, {} as HardhatConfigNetworks);
-}
-
-export function hardhatEtherscan(): HardhatEtherscanNetworks {
-  return CHAINS.reduce((etherscan, chain) => {
-    if (!chain.explorer || !chain.explorer.api) {
-      return etherscan;
-    }
-
-    const apiKey = chain.explorer.api.key;
-    const explorer = chain.explorer;
-    const apiKeyValue = apiKey.required ? chain.alias : "DUMMY_VALUE";
-
-    if (apiKey.hardhatEtherscanAlias) {
-      etherscan.apiKey[apiKey.hardhatEtherscanAlias] = apiKeyValue;
-      return etherscan;
-    }
-
-    etherscan.customChains.push({
-      network: chain.alias,
-      chainId: Number(chain.id),
-      urls: {
-        apiURL: explorer.api!.url,
-        browserURL: explorer.browserUrl,
-      },
-    });
-    etherscan.apiKey[chain.alias] = apiKeyValue;
-
-    return etherscan;
-  }, { apiKey: {}, customChains: [] } as HardhatEtherscanNetworks);
-}
+export * as hardhat from './hardhat';
+export * from './types';
 
 export function getEnvVariables(): string[] {
```

Comment:
My understanding is that this function is also hardhat-specific. So we can have a single `hardhat.getEnvVariables()` that has `MNEMONIC` pre-added and remove this function.

Reply:
Also maybe call it `getEnvVariableNames()`